<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="Buscar_n">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>560</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Filas:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_tabla">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Col">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="num_buscar">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>360</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>190</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>160</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>200</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Columnas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>330</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese el numero del cual quierer saber su posicion: </string>
      </property>
     </widget>
     <widget class="QLabel" name="mostrar_info2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>765</x>
        <y>190</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>FIlas/Columnas: </string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tW_Mostrar">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>561</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rotar_matriz">
      <property name="geometry">
       <rect>
        <x>835</x>
        <y>350</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotar matriz</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sB_Fil_Col">
      <property name="geometry">
       <rect>
        <x>885</x>
        <y>190</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QTableView" name="tableWidgetMatriz1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableWidgetMatriz2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableWidgetResultado">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSumar">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SUMA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRestar">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>RESTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMultiplicar">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>MULTIPLICACIÃ“N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Matriz A:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Matriz B:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxFilas1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxColumnas1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>510</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>420</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fila:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>490</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>490</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>420</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fila:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxFilas2">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>450</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxColumnas2">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>520</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonMostrarMatrices">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>MOSTRAR MATRICES</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>130</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>EJERCICIO #1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>330</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>EJERCICIO #3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>230</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>EJERCICIO #2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>EJERCICIO #4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>640</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CERRAR EL PROGRAMA</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
